sql.sql

```diff
@@ -0,0 +1,67 @@
+-- 1-Find out how many tasks are in the task table
```

Comment:
Nice Job! I would ask you to put the ; -semi colon- at the end of each query. it's not required when you are writing only one query. however, when you are writing multiple queries it's recommended to use it.